easybuild/easyconfigs/b/Blender/Blender-2.66a-foss-2016.04-Python-3.3.2.eb

```diff
@@ -21,17 +21,12 @@ configopts += '-DWITH_GAMEENGINE=OFF '
 configopts += '-DWITH_CYCLES=OFF '
 configopts += '-DWITH_SYSTEM_GLEW=OFF '
 
-# Python paths
-configopts += '-DPYTHON_LIBRARY=${EBROOTPYTHON}/lib '
-configopts += '-DPYTHON_INCLUDE_DIR=${EBROOTPYTHON}/include/python3.5m '
```

Comment:
@AFCJamie why did you drop these? They look relevant, but may need some tweaking, e.g. you could replace the hardcoded `3.5` with `%(pyshortver)s`